packages/react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModule.mm

```diff
@@ -219,7 +219,10 @@ id convertJSIValueToObjCObject(jsi::Runtime &runtime, const jsi::Value &value, s
   NSString *message = jsErrorDetails[@"message"];
 
   auto jsError = createJSRuntimeError(runtime, [message UTF8String]);
-  jsError.asObject(runtime).setProperty(runtime, "cause", convertObjCObjectToJSIValue(runtime, jsErrorDetails));
+  for (NSString* key in jsErrorDetails) {
```

Comment:
out of curiosity, what happens if we add a line like:
```objc
jsError.asObject(runtime).setProperty(runtime, [@"message" UTF8String], convertObjCObjectToJSIValue(runtime, message))
```
🤔 this might duplicate the `message` field in the own property (so it should be skipped as there is already another message field), but it should also add the `message` to the enumerable ones...